backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/importfile/BankStatementImportFileService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * #%L
+ * de.metas.banking.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.banking.importfile;
+
+import lombok.NonNull;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BankStatementImportFileService
+{
+	private final BankStatementImportFileRepository bankStatementImportFileRepository;
+
+	public BankStatementImportFileService(@NonNull final BankStatementImportFileRepository bankStatementImportFileRepository)
+	{
+		this.bankStatementImportFileRepository = bankStatementImportFileRepository;
+	}
+
+	@NonNull
+	public BankStatementImportFile getById(@NonNull final BankStatementImportFileId bankStatementImportFileId)
+	{
+		return bankStatementImportFileRepository.getById(bankStatementImportFileId);
+	}
+
+	public void processRecord(@NonNull final BankStatementImportFileId bankStatementImportFileId)
```

Comment:
please rename to setProcessed or markAsProcessed.
processRecord implied that the actuall processing takes place in here